.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+Fixes #
+
+## Proposed Changes
+
+  -
```

Comment:
is this markdown? why not `*`?

Reply:
I don't know :D https://www.markdownguide.org/cheat-sheet/ 
do you think it might be better ?

Reply:
Commonmark, the only one and true Markdown, says that both are ok. Also `+` can be used. I did not know that. https://spec.commonmark.org/0.29/#list-items